Querystring operator for objectbrowser (#80)
plone/plone.volto: README.rst

sure your add-on's profiles are applied AFTER this one.**

Queryparser operation for ObjectBrowser
---------------------------------------

> There is a new operation "*plone.app.querystring.operation.string.objectBrowserReference*" for path index in querystring.

Reviewer: ```suggestion
There is a new operation `plone.app.querystring.operation.string.objectBrowserReference` for path index in `querystring`.
```

Reviewer 2: I would skip "new", since it is old/normal soon.

> This is needed for pull-request `#3342 <https://github.com/plone/volto/pull/3342>`_  in Volto that enable the ObjectBrowser widget

Reviewer: ```suggestion
This is needed for pull request `#3342 <https://github.com/plone/volto/pull/3342>`_  in Volto that enables the `ObjectBrowser` widget
```

in listing blocks path criteria to select a path based on its UID and not a static path name.

> Expected data stored in "v" value of the criteria should be like this::

Reviewer: ```suggestion
Expected data stored in `v` value of the criteria should be similar to the following.

.. code-block:: python
```

  {
      "object": {
          "UID": "123456789",
          "Title": "Foo",
          "path": "/foo"
      },
      "depth": 1

> 

Reviewer: ```suggestion
```

  }

> 

Reviewer: Perhaps use 4-space indentation consistently above, including the initial indentation within `code-block` directive, assuming it is Python.
